docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -170,87 +149,225 @@ correct Google Workspace plan.
 
 ### Create an OAuth client ID
 
-In the [Create an OAuth client ID](https://console.cloud.google.com/apis/credentials/oauthclient) page of the GCP console, select "Web application" as the Application type, pick a name, then add the following as an authorized redirect URI.
+Under **Credentials**, select **CREATE CREDENTIALS**, then **OAuth client ID**:
 
-```txt
-https://<address of proxy server>:3080/v1/webapi/oidc/callback
-```
+![Create an OAuth Client ID](../../../img/sso/googleoidc/create-oauth-client-id.png)
+
+Select "Web application" as the Application type, pick a name, then set the
+redirect URI based on the path to your Teleport proxy or cloud tenant:
```

Comment:
```suggestion
redirect URI based on the address of your Teleport Proxy Service or cloud tenant:
```
There are a lot of proxies in a Teleport setup, so suggesting being more specific re: the Proxy Service.